src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/SdkDirectoryWorkloadManifestProvider.cs

```diff
@@ -118,7 +124,7 @@ public IEnumerable<ReadableWorkloadManifest> GetManifests()
         public IEnumerable<string> GetManifestDirectories()
```

Comment:
Should we consider adding the feature band as a parameter to this function? I think that might break the concept of some feature bands not touching others, so perhaps not. But for cleaning these manifests for unused bands I feel like we'll eventually add that.

Reply:
Right now it's not needed, the only external use of this method is for the resolver to find the `WorkloadManifest.targets` files in the manifests.

This manifest provider is specific to an SDK version, so if we need the data for other feature bands we might need to create a separate instance for each of them.  We can deal with those changes when we need to make them for clean.